test/vm.install

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -eu
+
+rpm -e lorax lorax-templates-generic lorax-lmc-virt lorax-lmc-novirt lorax-composer composer-cli || true
```

Comment:
For composer we'll need the ability to install an arbitrary list of RPMs (URLs to RPMs more precisely). This will facilitate testing pre-built packages from Brew/RHEL snapshots and also allow upgrading/installing other packages as necessary - for example anaconda RPMs to test changes in anaconda before they are merged. 

At the moment Jenkins supports the `EXTRA_RPMS` variable which contains all of these URLs and the system provisioning takes care to download them and install them onto the system before testing.

Reply:
`vm.install` installs the rpms that were just built in the source directory. We could add something like `EXTRA_RPMS` here, but I'm afraid that conflating these two use cases in one script might make things complicated (for example, we must ensure that the right versions of the packages are installed). It might be simpler to have this in two scripts.